src/index.html

```
@@ -104,18 +104,112 @@
-          autoplay
-        ></lottie-player>
+        <div class="loader-spinner">
+          <div class="spinner-ring"></div>
+          <div class="spinner-ring spinner-ring-2"></div>
+          <div class="spinner-dot"></div>
+        </div>
+        <div class="loader-text">CoIMS</div>
+        <div class="loader-subtext">Loading your workspace&hellip;</div>
+        <div class="loader-bar"><div class="loader-bar-fill"></div></div>
       </div>
+      <style>
```

HTMLHint (1.9.2)

[warning] The <style> tag cannot be used.

(style-disabled)